src/pages/workspaces/workspace/WorkspaceMenu.ts

```diff
@@ -85,7 +85,7 @@ interface DynamicWorkspaceMenuContentProps {
 /**
  * DynamicWorkspaceInfo is invoked when the name/namespace is passed instead of the derived states.
  * This happens from the list component, which also needs the workspace policies and bucketName for
- * sharing and cloning the workspace.
+ * sharing and cloning the workspace. This is also leveraged to pass the full decription during cloning as well.
```

Comment:
Are there other scenarios other than cloning where this could happen? Jc.

Reply:
not that I can think of - basically the reason why the description is cut off at all is because the list of workspaces only shows the firs t250 chars of description since that is all the view allows. the other options in the view are sharing (which doesnt include showing the full description and then leave, lock, delete, which also dont include it)

![image](https://github.com/DataBiosphere/terra-ui/assets/9418602/9ed23f47-9847-4018-a644-0c8fa501826a)